pkg/apis/pipeline/v1/task_validation.go

```diff
@@ -404,45 +402,6 @@ func validateObjectUsage(ctx context.Context, steps []Step, params []ParamSpec)
 	return errs.Also(validateObjectUsageAsWhole(steps, "params", objectParameterNames))
 }
 
-// validateObjectDefault validates the keys of all the object params within a
-// slice of ParamSpecs are provided in default iff the default section is provided.
-func validateObjectDefault(objectParams []ParamSpec) (errs *apis.FieldError) {
-	for _, p := range objectParams {
-		errs = errs.Also(ValidateObjectKeys(p.Properties, p.Default).ViaField(p.Name))
-	}
-	return errs
-}
-
-// ValidateObjectKeys validates if object keys defined in properties are all provided in its value provider iff the provider is not nil.
-func ValidateObjectKeys(properties map[string]PropertySpec, propertiesProvider *ParamValue) (errs *apis.FieldError) {
```

Comment:
Ah can you leave this function as is? I'm planning to use it in Chains. I think initially you are making this function public because I need to do keys comparisons in Chains, is it?

Reply:
Oh NVM because you are not using this function anymore in Chains, I will create this one in Chains then. Please ignore the previous commnent